handout/http/catching-rejections/cancel_request.md

```diff
@@ -0,0 +1,25 @@
+## Cancel a Request
+
+Cancelling an HTTP request is a common requirement. For example, you could have a queue of requests where a new request supersedes a pending request and that pending request needs to be cancelled.
+
+To cancel a request we call the `unsubscribe` function of its subscription.
+
+```ts
+@Component({ ... })
+export class MyApp {
+	...
+
+	search() {
+		const request = this.searchService.search(this.searchField.value)
+		  .subscribe(
+		    (result) => { this.result = result.artists.items; },
+		    (err) => { this.errorMessage = err.message; },
+        () => { console.log('Completed'); }
+		  );
+
+		request.unsubscribe();
```

Comment:
ok now I see what you are trying to do, you want to wait for a vm turn to prove that after the subscription was performed (moved to the event queue) you can unsubscribe from the observable. `setTimeout` makes sense.